src/blocks.zig

```diff
@@ -123,7 +127,17 @@ pub fn register(_: []const u8, id: []const u8, zon: ZonElement) u16 {
 	_hasBackFace[size] = zon.get(bool, "hasBackFace", false);
 	_friction[size] = zon.get(f32, "friction", 20);
 	_allowOres[size] = zon.get(bool, "allowOres", false);
-	_touchFunction[size] = TouchFunctions.getFunctionPointer(zon.get([]const u8, "touchFunction", ""));
+	_tickEvent[size] = TickEvent.loadFromZon(zon.getChild("tickEvent"));
+
+	const touchFunctionName = zon.get([]const u8, "touchFunction", "");
```

Comment:
Instead of using `""` to signify that the zon entry did not exist, better use an optional.
This also gets rid of the weird and confusing error handling paths.
```suggestion
	const touchFunctionName = zon.get(?[]const u8, "touchFunction", null);
```